### PR TITLE
Adding outside of england to screener

### PR DESCRIPTION
**Pull request by cjrace**
Added it into the acceptable regions csv, and also some test data along with a unit test for it.

### CODE REVIEW

File: data/regions.csv [hunk 1]
```diff
@@ -10,3 +10,4 @@ E12000008,South East
 E12000009,South West
 E13000001,Inner London
 E13000002,Outer London
+z,Outside of England and unknown
```
Comment (lauraselby) [inline data/regions.csv:13]: Are we really expecting the label to be 'Outside of England and unknown'? Should they not be split and just both share the z code?

Comment (cjrace) [reply data/regions.csv:13]: Yes, that is the specific aggregation they have always used with the name of 'Other'. Even if they did split it up (which we discussed) they should still supply the combination of them all as users expect to see it.

**Review by sarahmwong — DISMISSED** [review]

**Review by sarahmwong — COMMENTED** [review]
Sorry know I just approved a sec ago! Just ran through on dev with some other names for "z" and it passes all checks is that an issue for EES if things share a code?
image

**Comment by cjrace** [comment]
@sarahmwong very good spot! Though I suspect this may already have been an issue before this PR, can you test that file on master too? We control Not available quite closely, but haven't yet put any similar limits onto Not applicable (off the top of my head). 

If it's as I suspect then I'll stick something on trello as we should probably treat them both the same, and then add additional things in (like the one in this PR) by exception.

**Comment by sarahmwong** [comment]
> @sarahmwong very good spot! Though I suspect this may already have been an issue before this PR, can you test that file on master too? We control Not available quite closely, but haven't yet put any similar limits onto Not applicable (off the top of my head).
> 
> If it's as I suspect then I'll stick something on trello as we should probably treat them both the same, and then add additional things in (like the one in this PR) by exception.

Ah yes same thing happening on master - good plan! Will approve this PR for now as it will match the guidance PR :)

**Review by sarahmwong — APPROVED** [review]

**Review by lauraselby — APPROVED** [review]

**Pull request merged**